src/chains/ethereum/transaction/src/access-lists.ts

```diff
@@ -43,7 +43,7 @@ export class AccessLists {
       const json: AccessList = [];
       for (let i = 0; i < bufferAccessList.length; i++) {
         const data = bufferAccessList[i];
-        const address = Data.from(data[0], 32).toString();
+        const address = Data.from(data[0], 20).toString();
```

Comment:
It won't let me add a comment to the line (I guess cause it's too far from the line that actually changed?) but line 30 above needs this change as well. This is for the case where we are creating an address buffer from an address string, we also are incorrectly padding to 32 bytes. Great find!

Reply:
followed-up to fix that one too, thanks!